data/ShivangShandilya.json

```diff
@@ -2,73 +2,119 @@
   "name": "Shivang Shandilya",
   "displayStatsPublic": true,
   "type": "personal",
-  "bio": "DevOps | Open Source | Front - End Developer | DSA",
+  "bio": "**DevOps** | **Open Source** | Front-End Developer | DSA",
+  "tags": ["Open Source", "DevOps", "Docker", "Kubernetes", "Go", "AWS"],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/shivv_twt" },
+    { "icon": "FaGithub", "url": "https://github.com/ShivangShandilya" },
+    { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/shivang-shandilya-8b926b218/" },
+    { "icon": "FaInstagram", "url": "https://www.instagram.com/shivangshandilyaa/" }
+  ],
 
   "links": [
     {
-      "name": "Follow me on Twitter",
+      "group": "Socials",
+      "name": "Daily tweets on DevOps and Open Source",
       "url": "https://twitter.com/shivv_twt",
       "icon": "FaTwitter"
     },
     {
-      "name": "Follow me on LinkedIn",
+      "group": "Socials",
+      "name": "GitHub - My Open Source Profile",
+      "url": "https://github.com/ShivangShandilya",
+      "icon": "FaGithub"
+    },
+    {
+      "group": "Socials",
+      "name": "Let's connect on LinkedIn",
       "url": "https://www.linkedin.com/in/shivang-shandilya-8b926b218/",
       "icon": "FaLinkedin"
     },
     {
-      "name": "Follow me on GitHub",
-      "url": "https://github.com/ShivangShandilya",
-      "icon": "FaGithub"
+      "group": "Socials",
+      "name": "Instagram - Just Connect, why thinking?",
+      "url": "https://www.instagram.com/pradumnasaraf",
```

Comment:
Please check the link to your instagram as this does not appear to be your account

Reply:
Oops, I was using Pradumna's Linkfree file as a reference looks like I forgot to change the links. Thanks for pointing this out 😁😁

Reply:
Issue has been resolved, do tell me if there is anything else that needs to be corrected 😁